scrapers_next/tx/committees.py

```diff
@@ -0,0 +1,103 @@
+import re
+from spatula import HtmlPage, HtmlListPage, CSS, XPath, SelectorError
+from openstates.models import ScrapeCommittee
+
+
+class CommitteeDetail(HtmlPage):
+    example_source = "https://capitol.texas.gov/Committees/MembershipCmte.aspx?LegSess=87R&CmteCode=C014"
+
+    def process_page(self):
+        com = self.input
+        com.add_source(self.source.url)
+
+        table = XPath("//div[@id='content']/table[2]/tr[position()>1]").match(self.root)
+        for person in table:
+            name = person.text_content()
+
+            positions = ["chair", "vice chair"]
+            if name:
+                try:
+                    label_text = XPath("./td[1]//text()").match(person)
+                    for label in label_text:
+                        label = label.lower()
+                        if label.endswith(":"):
+                            label = label[:-1]
+                        if label in positions:
+                            role = label
+                        else:
+                            role = "member"
+                    name = person.text_content().split(":")[1]
+                except SelectorError:
+                    role = "member"
+
+            # there are two spaces in between each name
+            name = (
+                name.replace("Sen.", "").replace("Rep.", "").replace("  ", " ").strip()
+            )
+            com.add_member(name, role)
+
+        # extra information
+        table = XPath("//div[@id='content']/table[1]/tr/td//text()").match(self.root)
+        table = [
+            thing.replace("\r", "")
+            .replace("\n", "")
+            .replace("\t", "")
+            .replace(": ", "")
+            for thing in table
+        ]
+
+        fields = table[1::2]
+        extra = table[2::2]
+
+        for i in range(5):
```

Comment:
a good rule of thumb is to always explain "magic numbers" like this in a comment or variable name, why is it 5?